sections/php.zsh

```diff
@@ -27,11 +27,11 @@ spaceship_php() {
 
   spaceship::exists php || return
 
-  local php_version=$(php -v 2>&1 | grep --color=never -oe "^PHP\s*[0-9.]*" | awk '{print $2}')
+  local php_version=$(php -v 2>&1 | grep --color=never -oe "^PHP\s*[0-9.]\+" | awk '{print $2}')
 
   spaceship::section \
     "$SPACESHIP_PHP_COLOR" \
     "$SPACESHIP_PHP_PREFIX" \
-    "${SPACESHIP_PHP_SYMBOL}v${php_version}" \
+    "${SPACESHIP_PHP_SYMBOL} v${php_version}" \
```

Comment:
This is not needed. An empty space is defined with variable.

https://github.com/denysdovhan/spaceship-prompt/blob/5a391a571a0a05a038d880e7359228f6c3525511/sections/php.zsh#L14

This might have to do with your [terminal unicode width settings](https://github.com/denysdovhan/spaceship-prompt/blob/master/docs/Troubleshooting.md#some-section-icons-overlap-each-other)